openedx/features/content_tagging/rules.py

```diff
@@ -43,7 +67,7 @@ def can_view_taxonomy(user: User, taxonomy: oel_tagging.Taxonomy = None) -> bool
     """
     if taxonomy:
         taxonomy = taxonomy.cast()
-    return (taxonomy and taxonomy.enabled) or is_taxonomy_admin(user, taxonomy)
+    return not taxonomy or taxonomy.enabled or is_taxonomy_admin(user)
```

Comment:
The exact change we did in openedx-learning. I think this is the cause of the errors in test_rules.py. I will look into it tomorrow.